charts/terra-argocd-app/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: terra-argocd-app
 description: A Helm chart for generating ArgoCD resources for a Terra app in an environment
 type: application
-version: 0.1.2
+version: 0.1.3
```

Comment:
Don't need to bump version anymore, if you really want a patch version you can comment #patch in the merge commit

Reply:
derp, sorry, keep forgetting, thx!

Reply:
All good, it's NBD